demo-plugin/src/main/kotlin/org/jetbrains/research/refactoringDemoPlugin/modelInference/license/Apache_2_0.kt

```diff
@@ -0,0 +1,69 @@
+package org.jetbrains.research.refactoringDemoPlugin.modelInference.license
+
+object Apache_2_0 : License {
+    override val name: String = "Apache License 2.0"
+
+    override val fullText: String =
```

Comment:
Why not store the full text in the `resources`, like we do with other data?

Reply:
Good point! I suggest fixing it in the separate PR after merging this PR and Visualization information PR because only last one contains the code responsible for the plugin resource bundle (storing text in the properties file in the resources folder).